events/index.ts

```diff
@@ -0,0 +1,7 @@
+import { InteractionCreateEvent } from './interactionCreate';
+import { ReadyEvent } from './ready';
+
+export const AllEvents = [
```

Comment:
It's not directly related to your changes, but in most discord codebases I've seen, events are directly set up with the `.on` property, so something like `client.on('ready')`, I found that to be a lot clearer to read instead of going the file route. What are your thoughts?

Reply:
I will have a go at moving to the explicit registration approach. I think that with the number of events we are currently handling, doing an explicit registration might be simpler and clearer. We can then look at going back to this more dynamic approach if we end up with a larger number of events in the future.

Reply:
Agreed!

Reply:
I have taken the explicit registration approach in the latest commit and I think it looks ok.